Description: Don't run dynamic_lib::tests::test_loading_cosine on Aarch64 and
 powerpc64le whilst it fails.
Author: Chris Coulson <chris.coulson@canonical.com>
Bug: https://github.com/rust-lang/rust/issues/45410

--- a/src/librustc_metadata/dynamic_lib.rs
+++ b/src/librustc_metadata/dynamic_lib.rs
@@ -91,7 +91,7 @@
 
     #[test]
     fn test_loading_cosine() {
-        if cfg!(windows) {
+        if cfg!(any(windows, target_arch = "aarch64", all(target_arch = "powerpc64", target_endian = "little"))) {
             return
         }
 
